### PR TITLE
fix: [ui] fix search filter background

### PR DESCRIPTION
**Pull request by pppanghu77**
fix search filter background and fix tag crumbedit for refresh will lose text when editing

Log: [ui] fix search filter background
Bug: https://pms.uniontech.com/bug-view-237399.html https://pms.uniontech.com/bug-view-237399.html

### CODE REVIEW

File: src/plugins/filemanager/core/dfmplugin-sidebar/treeviews/sidebarwidget.cpp [hunk 1]
```diff
@@ -289,6 +290,12 @@ void SideBarWidget::onItemRenamed(const QModelIndex &index, const QString &newNa
 
 void SideBarWidget::initializeUi()
 {
+    QGraphicsDropShadowEffect *effect = new QGraphicsDropShadowEffect;
```
Comment (Kakueeen) [inline src/plugins/filemanager/core/dfmplugin-sidebar/treeviews/sidebarwidget.cpp]: 没有父对象，能正常析构吗

File: src/plugins/filemanager/dfmplugin-search/topwidget/advancesearchbar.cpp [hunk 2]
```diff
@@ -445,17 +450,21 @@ AdvanceSearchBar::AdvanceSearchBar(QWidget *parent)
     : QScrollArea(parent),
       d(new AdvanceSearchBarPrivate(this))
 {
+    QGraphicsDropShadowEffect *effect = new QGraphicsDropShadowEffect;
```
Comment (Kakueeen) [inline src/plugins/filemanager/dfmplugin-search/topwidget/advancesearchbar.cpp]: 同上

**Review by Kakueeen — APPROVED** [review]

File: src/plugins/filemanager/dfmplugin-search/topwidget/advancesearchbar.cpp (hunk 2 above)
Comment (pppanghu77) [reply src/plugins/filemanager/dfmplugin-search/topwidget/advancesearchbar.cpp]: 已修复更新

File: src/plugins/filemanager/core/dfmplugin-sidebar/treeviews/sidebarwidget.cpp (hunk 1 above)
Comment (pppanghu77) [reply src/plugins/filemanager/core/dfmplugin-sidebar/treeviews/sidebarwidget.cpp]: 已修复更新

**Review by max-lvs — APPROVED** [review]

**Comment by pppanghu77** [comment]
merge

**Pull request merged**